README.md

```diff
@@ -70,10 +70,10 @@ You can choose list`s type.
 
 ## Output data
 
-| Field | Type      | Description                              |
-| ----- | --------- | ---------------------------------------- |
-| style | `string`  | type of a list: `ordered` or `unordered` |
-| items | `Item[]`  | the array of list's items                |
+| Field | Type              | Description                              |
+| ----- | ----------------- | ---------------------------------------- |
+| style | `string`          | type of a list: `ordered` or `unordered` |
+| items | `(Item \| string)[]` | the array of list's items                |
```

Comment:
Let me know whether I should keep this README or if there is another way you prefer to document it. 

The current README change feels incorrect because this is included in "Output data" but the data is never saved with `items: string[]`.

Reply:
I think, we can remove it at all, because it's just a legacy format support, it shouldn't be mentioned in main Readme

Reply:
Also, please increment a patch version